x/merkledb/db.go

```diff
@@ -34,19 +35,21 @@ const (
 	DefaultEvictionBatchSize = 100
 	RootPath                 = EmptyPath
 	// TODO: name better
-	rebuildViewSizeFractionOfCacheSize = 50
-	minRebuildViewSizePerCommit        = 1000
+	rebuildViewSizeFractionOfCacheSize   = 50
+	minRebuildViewSizePerCommit          = 1000
+	rebuildIntermediateDeletionWriteSize = 10 * units.KiB
```

Comment:
What is a reasonable value for this?

Reply:
used [here](https://github.com/ava-labs/avalanchego/pull/1918/files#diff-082390fa82b4b107f83a66d137350efd78b821414dcc89708f2ec45d20af89aaR273)

Reply:
Maybe 1 units.MiB?